setup.py

```diff
@@ -27,20 +27,20 @@ def readme():
     author='Colin Curtain',
     author_email='ccbogel@hotmail.com',
     license='MIT',
-    install_requires=[
-        'graphviz',
-        'pyqt5',
-        'lxml',
-        'Pillow', 
-        'pikepdf', 
-        'six', 
-        'ebooklib',
-        'pdfminer.six',
-        'ply',
-        'chardet',
-        'python-vlc',
-        'click',
-    ],
+    # install_requires=[
```

Comment:
why are these modules commented out?

Reply:
this would need some playing around with dh-virtualenv